src/exampleTypes/ResultTable/Header.js

```diff
@@ -157,7 +166,12 @@ let Header = ({
           Remove Column
         </DropdownItem>
         {!!addOptions.length && (
-          <DropdownItem onClick={F.on(adding)}>
+          <DropdownItem
+            onClick={() => {
+              setAddColumnTriggerField(field)
```

Comment:
Why is this state necessary? You can access `field` in the nested picker

Reply:
`field` coming from the nested picker is the field the user selected. That is not the field he clicked on (header cell) that we need to remember so we know where to place the new column. Not sure we are on the same page here.

Reply:
Yeah but you can just rename the `field` variable in the nested picker to something else and you'll have access to both

Reply:
Got it. Make sense. Thanks